### ISSUE TITLE
NTR: Lens thick (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7987]
New term request Lens thick (elthat_130401043445):

**Comment by pnrobinson** [comment]
@MickeySegal
Is this related to [Lenticonus HP:0001142](https://hpo.jax.org/app/browse/term/HP:0001142) ?

**Comment by MickeySegal** [comment]
It seems to be separate.  As an example, https://www.omim.org/entry/613517 lists "features include an essentially normal anterior segment, steep corneal curvatures, shallow anterior chamber, thick lenses, and thickened scleral wall".

**Comment by pnrobinson** [comment]
@MickeySegal
I am not convinced this is a thing. The overall eye is smaller but apparently the lens retains (more of) its normal size, resulting in a high lens/eye volume ratio. There is only one mention of thickened lens without much detail, so this may just have been a subjective impression. This is probably mainly Microphthalmia -- ? Unless there is a really clear distinction, I think we can probably do without this term?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1459/ "Eye findings previously associated with Fryns syndrome have included central/paracentral corneal clouding that may result from abnormal corneal endothelium, microphthalmia, irregularities of Bowman's layer, **thickened posterior lens capsule**, and retinal dysplasia. In those with PIGN pathogenic variants, cloudy corneas and cataracts have been described."

So the term seems to be used even in the absence of microphthalmia.

**Comment by pnrobinson** [comment]
@daboe1  Could you advise on this term?
In [Microphthalmia, isolated 6](https://www.omim.org/entry/613517), 
[Gal et al](https://pubmed.ncbi.nlm.nih.gov/21397065/) speak of thickened lens and state that there is a high lens/eye volume ratio (mean of 29%).

Can we define this as
**Label**: Increased anteroposterior lens thickness
**Definition**: Increase in the anteroposterior (axial) dimension of the crystalline lens relative to the overall length of the globe.
PMID: 21397065

**Comment by pnrobinson** [comment]
@MickeySegal  I think the above is correct.
adding as 
HP:0020308
Label: Increased anteroposterior lens thickness
Definition: Increase in the anteroposterior (axial) dimension of the crystalline lens relative to the overall length of the globe.
